spec/spec_helper.rb

```diff
@@ -7,8 +7,8 @@
 require 'onlyoffice_documentserver_conversion_helper'
 require 'onlyoffice_logger_helper/logger_helper'
 require_relative '../config/StaticData'
-require_relative '../helpers/PalladiumHelper'
-require_relative '../helpers/DocumentServerHelper'
+require_relative '../helpers/palladium_helper'
```

Comment:
I'll try to place require files in alphabetical order, so `document_server_helper` will be before `palladium_helper`

I think you can safely change all orders in that file or create new PR, as you wish